griptape/templates/engines/eval/results/system.j2

```diff
@@ -0,0 +1,10 @@
+Given the evaluation steps, return a JSON with two keys:
+1) a `score` key ranging from 0 - 10, with 10 being that it follows the criteria outlined in the steps and 0 being that it does not.
```

Comment:
Why not `0.0` - `1.0`? and avoid the division afterwords? (which is temporally coupled to this specific prompt template)

Reply:
Answer [here](https://github.com/griptape-ai/griptape/pull/1396#discussion_r1889012269)